package.json

```diff
@@ -32,32 +32,45 @@
   "scripts": {
     "cover": "nyc --reporter=lcov mocha test/*.js && nyc report",
```

Comment:
No need to have this. Also, let's remove `nyc` from everywhere in `package.json`, we use `c8` instead.

Reply:
sure, after removing this one, no `nyc` in the package json as all